packages/nimble-storybook/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@ni-private/nimble-storybook",
+  "version": "1.0.0",
+  "private": true,
+  "type": "module",
+  "description": "Storybook for the Nimble repo",
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json && storybook build -o dist/storybook --webpack-stats-json",
+    "start": "storybook dev -p 6006",
+    "lint": "eslint .",
+    "format": "eslint . --fix"
+  },
+  "dependencies": {
```

Comment:
I'm not sure the distinction between dev and devDep is meaningful here. Lets just move these all to devDep